requirements-docs.txt

```diff
@@ -2,3 +2,6 @@ sphinx==5.3.0
 furo==2022.12.7
 sphinx_copybutton==0.5.1
 sphinx-remove-toctrees==0.0.3
+# Matches the urllib3 requirements for botocore.
+urllib3>=1.25.4,<1.27 ; python_version < "3.10"
+urllib3>=1.25.4,!=2.2.0,<3 ; python_version >= "3.10"
```

Comment:
If we're going to add these, I think we may want to simplify it to the strict requirements. Otherwise, we may end up with odd behaviors as these change and I'd rather we not have to worry about keeping this in-sync in more places.

```suggestion
# Avoid urllib3 2.x below Python 3.10
urllib3<2.0 ; python_version < "3.10"
```